src/files/utils/yum.ts

```diff
@@ -59,7 +60,7 @@ const signRpm = async (rpm: string) => {
     }
     const keyId = keyMatch[1];
     // Sign the RPM file
-    const [exe, args] = getSignRpmCommand(tmpDir, ['-D', `"_gpg_name ${keyId}"`, '--addsign', path.basename(rpm)]);
+    const [exe, args] = getSignRpmCommand(tmpDir, ['-D', '_gpg_name ${keyId}', '--addsign', path.basename(rpm)]);
```

Comment:
These still need to be backticks

Reply:
sorry, that's what I get for running a different version than I'm committing. Lession learned, I guess